lib/thermometer.js

```diff
@@ -880,7 +880,7 @@ Controllers.DEFAULT = Controllers.ANALOG;
 
 var priv = new Map();
 
-class Thermometer extends Withinable {
+class Thermometer extends Emitter {
```

Comment:
I'm confused on this point. Thermometers should extend Withanble right?

Reply:
This is a mistake!

Reply:
...but I can't find this code 0_0

Reply:
In my local it's `class Thermometer extends Withinable {`

Reply:
Derp, I was on a different branch. Fix coming.